bundles/org.openhab.binding.openweathermap/README.md

```diff
@@ -389,16 +389,15 @@ Number:Temperature localTodayEveningApparent "Evening apparent Temperature [%.1f
 Number:Temperature localTodayNightApparent "Night apparent Temperature [%.1f %unit%]"  <temperature> { channel="openweathermap:onecall:api:local:forecastToday#apparent-night" }
 Number:Pressure localTodayPressure  "Barometric pressure [%.1f %unit%]"  <pressure> { channel="openweathermap:onecall:api:local:forecastToday#pressure" }
 Number:Dimensionless localTodayHumidity  "Atmospheric humidity [%d %unit%]"  <humidity> { channel="openweathermap:onecall:api:local:forecastToday#humidity" }
-Number:Temperature localTodayDewpoint "Dew point [%.1f %unit%]"  <Temperature> { channea="openweathermap:onecall:api:local:forecastToday#dew-point" }
+Number:Temperature localTodayDewpoint "Dew point [%.1f %unit%]"  <Temperature> { channel="openweathermap:onecall:api:local:forecastToday#dew-point" }
 Number:Speed localTodayWindSpeed  "Wind speed [%.1f km/h]"   <wind> { channel="openweathermap:onecall:api:local:forecastToday#wind-speed" }
 Number:Angle localTodayWindDirection "Wind direction [%d %unit%]" <wind> { channel="openweathermap:onecall:api:local:forecastToday#wind-direction" }
 Number:Speed localTodayGustSpeed "Gust speed [%.1f km/h]" <wind> { channel="openweathermap:onecall:api:local:forecastToday#gust-speed" }
-Number:Dimensionless localTodayPrecipProbability  "Precipitation probability [%.1f]"    { channel="openweathermap:onecall:api:local:forecastToday#probability" }
```

Comment:
There is a small typo: Should be `precip-probability`.

Reply:
done